doc/README.NSSLmp

```diff
@@ -0,0 +1,162 @@
+Some background information and usage tips for the NSSL microphysics scheme.
+
+
+ IMPORTANT: Best results are attained using WENO (Weighted Essentially Non-Oscillatory) scalar advection option. This helps to limit oscillations at the edges of precipitation regions (i.e., sharp gradient), which in turns helps to prevent mismatches of moments that can show up as noisy reflectivity values.
+   moist_adv_opt  = 4,
+   scalar_adv_opt = 3,
+ The monotonic option (2) is less effective, but better than the default positive definite option (1)
+
+NOTE TO SMPAR or DM+SMPAR USERS: If a segmentation fault occurs, try setting the environment variable OMP_STACKSIZE to 8M or 16M (default is 4M, where M=MB). Note that this does not increase the shell stacksize limit [use 'ulimit -a unlimited' (bash) or 'unlimit stacksize' (tcsh)]
+
+CHANGES:
+June 2023 (WRF 4.5.X): Main default option change is for graupel/hail fall speed options (icdx, icdxhl; changed from 3 to 6, see below), and default maximum gr/hail droplet collection efficiencies (ehw0/ehlw0 changed from 0.5/0.75 to 0.9/0.9, see below)
```

Comment:
@MicroTed Can you change this to WRF 4.6?

Reply:
Yes, got that and other instances. I also made a few small updates to the text.

Reply:
Thanks! I think we are ready to accept the change.